packages/dso-toolkit/src/components/accordion/accordion.args.ts

```diff
@@ -19,15 +20,18 @@ export interface AccordionArgs {
   icon: string;
   heading: AccordionHeading;
   handleUrl: string;
-  handleTitle: string;
+  handleTitle: RenvooiValue;
+  handleTitleText: string;
   demoScrollIntoView: "start" | "end" | undefined;
   label: string;
   labelStatus: LabelStatus;
 }
 
-export const accordionArgs: Pick<AccordionArgs, "demoScrollIntoView" | "open"> = {
+export const accordionArgs: Pick<AccordionArgs, "demoScrollIntoView" | "open" | "handleTitle" | "handleTitleText"> = {
   open: false,
   demoScrollIntoView: undefined,
+  handleTitle: "ongewijzigd",
```

Comment:
Dit gaat goed omdat de term "ongewijzigd" een string is, dus aan het type "Renvooivalue" voldoet.

Ik vind Renvooi ook best knap lastig.

Ik denk dat we @tfrijsewijk even moeten inschakelen.